playwright/pages/shared/TypeBuilderPage.ts

```diff
@@ -285,6 +293,13 @@ export class TypeBuilderPage extends BuilderPage {
     ).toBeVisible();
   }
 
+  async saveUidLabel(label: string) {
+    await this.getUidEditorButton("UID").click();
+    await expect(this.uidEditorDialog).toBeVisible();
+    await this.page.getByRole("textbox", { name: "Label *" }).fill(label);
```

Comment:
⚠️ #issue

Please follow best practices: https://github.com/prismicio/slice-machine/tree/main/playwright#always-try-to-do-an-exact-matching-with-locators

Here for `Always try to do an exact matching with locators`